source/systems/request.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Request System
+layout: default
+---
+# Request System
+
+As the mayor of a fledgling colony, you are sure to be busy managing your citizens and the multitude of issues that arise. At some point, you’ll have too many citizens to deal with every single one of their demands individually.
+
+Thankfully, there’s the Request System! Citizens will automatically ask for the items they need to perform their jobs, and if those items can be provided by your other citizens, they will do so! 
+
+## Getting Started
+
+The request system is based on two components: the [Warehouse](../../source/buildings/warehouse) — which stores all of your colony’s resources — and the [Couriers](../../source/workers/courier) — who pick up and drop off items between buildings. The couriers wait in the warehouse until a request comes in.
```

Comment:
"courier" and "warehouse" should always be capitalized

Reply:
Done.